### PR TITLE
Add documentation for Flask deployment

### LINKED ISSUE #155: Add documentation for Flask deployment
Add instructions to the documentation on how to use the MovieBot when deployed with Flask.

### PR DESCRIPTION
**Pull request by NoB0**
Add documentation on how to run MovieBot with Flask. It is based on the modifications done in PR #175.

Fixes #155

### CODE REVIEW

File: docs/source/usage.rst [hunk 1]
```diff
@@ -24,6 +24,97 @@ To run MovieBot with default configuration execute the command below.
...
+        ChatWidget({
+            name: "MovieBot",
+            serverUrl: "http://127.0.0.1:5001",
+            socketioPath: "/socket.io/",
+            useFeedback: false,
+            useLogin: false,
+        });
+    </script>
+
+A button to discuss with MovieBot should be placed at the bottom right of your web page.
```
Comment (IKostric) [inline docs/source/usage.rst:116]: Nit: discuss seems a bit strange in this context.

**Review by IKostric — APPROVED** [review]
LGTM

File: docs/source/usage.rst (hunk 1 above)
Comment (NoB0) [reply docs/source/usage.rst:116]: Replaced by interact.

File: docs/source/usage.rst [hunk 2]
```diff
@@ -24,6 +24,97 @@ To run MovieBot with default configuration execute the command below.
...
+
+   # Start a new conversation
+   r = requests.post(
+            uri,
+            json={
+                     "message": {"text": "/start"},
+                     "sender": {"id": "test_user"}, 
+            },
+        )
+   print(r.json())
```
[the remark this answers is not in the document]
Comment (NoB0) [reply docs/source/usage.rst:88]: Done.

**Pull request merged**